content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/starshipit_shipping.rst

```diff
@@ -0,0 +1,201 @@
+shipping order to set the correct courier on your order. There are multiple ways to do so. We
+recommend adding a rule with the condition being ":guilabel:`contains`", the value being
+":guilabel:`the product code`", and the action "setting that product code and courier" to the
+shipping order.
+
+.. image:: starshipit_shipping/starshipit-rules.png
+   :align: center
+   :alt: Shipping rules in the Starshipit settings.
+
+Finding Starshipit API credentials
+----------------------------------
+
+In your Starshipit account, navigate to :menuselection:`Settings --> API` in the side menu.
+There, you can find the API keys you will need to connect to Odoo.
+
+.. image:: starshipit_shipping/starshipit-settings-api.png
+   :align: center
+   :alt: Finding the Starshipit API keys.
+
+Setup in Odoo
+=============
+
+Install the Starshipit shipping module
+--------------------------------------
+
+After the Starshipit account is set up and configured, it's time to configure the Odoo database.
+To get started, go to Odoo's :guilabel:`Apps` module,  search for the :guilabel:`Starshipit
+Shipping` integration, and install it.
+
+.. image:: starshipit_shipping/starshipit-app.png
+   :align: center
+   :alt: Starshipit Shipping module in the Odoo Apps module.
+
+Starshipit shipping connector configuration
+-------------------------------------------
+
+Once installed, activate the :guilabel:`Starshipit Shipping` module in :menuselection:`Inventory -->
+Configuration --> Settings`. The :guilabel:`Starshipit Connector` setting is found under the
+:guilabel:`Shipping Connectors` section.
+
+After activating the :guilabel:`Starshipit Connector`, click on the :guilabel:`Starshipit Shipping
+Methods` link below the listed connector. Once on the :guilabel:`Shipping Methods` page, click
+:guilabel:`Create`.
+
+.. tip::
+   :guilabel:`Shipping Methods` can also be accessed by going to :menuselection:`Inventory -->
+   Configuration --> Delivery --> Shipping Methods`.
+
+Fill out the following fields in the :guilabel:`New Shipping Method` form:
+
+- :guilabel:`Shipping Method`: type `Starshipit`.
+- :guilabel:`Provider`: select :guilabel:`Starshipit` from the drop-down menu.
+- :guilabel:`Delivery Product`: set the product that was configured for this shipping method or
+  create a new product.
+- In the :guilabel:`Starshipit Configuration` tab, enter the :guilabel:`Default Package Type`
+  if you are going to use packages during shipping in order to get more accurate rates.
+  The packages system may be enabled in the Odoo settings.
+- In the :guilabel:`Starshipit Configuration` tab, enter the :guilabel:`Origin Address`.
+  It will be used as sending address when getting available services for your integration.
+- In the :guilabel:`Starshipit Configuration` tab, enter the :guilabel:`Starshipit API Key`.
+- In the :guilabel:`Starshipit Configuration` tab, enter the :guilabel:`Starshipit Subscription Key`
```

Comment:
Reordered the items in this bulleted list to match the order in which they appear on the form in Odoo
```suggestion
In the :guilabel:`Starshipit Configuration` tab, fill out these fields:

- :guilabel:`Starshipit API Key`: enter the :abbr:`API (Application Programming Interface)` key
  :ref:`obtained from Starshipit <inventory/shipping_receiving/star-api>`.
- :guilabel:`Starshipit Subscription Key`: enter the subscription key obtained from the same place
  as the :ref:`API key <inventory/shipping_receiving/star-api>`.
- :guilabel:`Origin Address`: Enter the address where products are shipped from. This field is
   crucial for calculating shipping rates and :ref:`generating shipping labels
   <inventory/shipping_receiving/star-label>`.
- :guilabel:`Default Package Type`: Set a default package type to include the weight of the empty
   package when automatically calculating shipping rates.

.. important::
   To set a default package type, the *Packages* feature **must** be enabled in
   :menuselection:`Inventory --> Configuration --> Settings`.
```